### DISCUSSION TITLE
next/image lighthouse flags SVG images missing width and height

### DISCUSSION THREAD

**Discussion by ystewart** [vercel/next.js#20019] [Help]
Hello,

I am using next/image component for some SVG images and Lighthouse shows the "Image elements do not have explicit width and height" flag in the performance category. Has anyone else had this issue? and know a potential fix?

**Comment by nghiepnguyen520** [comment]
Next v10.0.0 support for optimization. You should be specific for Width and Height 

> `<Image
        src="/me.png"
        alt="Picture of the author"
        width={500}
        height={500}
      />`

**Reply by stephent** [reply]
Even if you include width and height attributes in the <Image> tag, it seems they don't always make it into the generated markup, which is what is triggering Lighthouse, e.g.:

```
<img alt="logo" src="/_next/image?url=%2Fimages%2Flogos%2Flogo.png&amp;w=384&amp;q=75" 
decoding="async" style="visibility: inherit; position: absolute; inset: 0px; box-sizing: border-box; padding: 0px; border: none; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;" 
srcset="/_next/image?url=%2Fimages%2Flogos%2Flogo.png&amp;w=256&amp;q=75 1x, /_next/image?url=%2Fimages%2Flogos%2Flogo.png&amp;w=384&amp;q=75 2x">
```
In this case, `width={140} height={45}` was set on the `<Image>` tag in original source.

Bug? Or Lighthouse limitation caused by not parsing `srcset`?

**Comment by michrome** [comment]
There's an open issue for this at https://github.com/vercel/next.js/issues/20060. The upshot is it's an **upstream issue** to be fixed in Lighthouse: https://github.com/GoogleChrome/lighthouse/issues/11631.
